src/sql/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -66,6 +66,11 @@ export class Checkbox extends Widget {
 			this.onChange(opts.onChange);
 		}
 
+		this._el.addEventListener('focus', () => {
+			this._el.style.outlineColor = '#E47609';
```

Comment:
1. Colors like this should be done through the color registry so they can be changed by themes. We should almost never hardcode colors like this.
1. There's no blur/focus out to remove the outline?